public/js/contact_info.js

```diff
@@ -0,0 +1,29 @@
+require([
+  'jquery', 'underscore', 'transport', 'auth', 'client-models', 'logger', 'bootstrap', 
+  'backbone.marionette'
+], function($, _, transport, auth, models, logging) {
+
+var views = {};
+var logger = new logging.Logger("contact-info");
+
+//Contact form view for filling preferred method of contact 
+views.ContactFormView = Backbone.Marionette.ItemView.extend({
+	
+	events: {
+		'click .submit-contact-form' : 'submitContactForm'
+	},
+
+	initialize: function(options) {
+		Backbone.Marionette.ItemView.prototype.initialize.call(this, options);
+	},
```

Comment:
Unless you are doing additional work in the constructor, you can remove this ``initialize`` override.  This code just says "Override the initialize function with what the initialize function would be if we didn't override it."